common/country_types/hfe_country_types.txt

```diff
@@ -231,13 +283,12 @@ hfe_drone = {
 		category = country_base
 		produces = {
 			influence = 3
+			minerals = 50
```

Comment:
Too cheat

Reply:
The Harvesting Marvel consists of Ancient MINING Drones. MINING, as in minerals. Therefore, they should have BUTT LOADS of Minerals coming in at all times.

Reply:
They have balanced starbase buildings and deposits in star system.
I dont want income from nowhere.

Reply:
Every country gets a base level of "income from nowhere".

Reply:
They have started modifier and started resources.
I do not see why it need also country base.
Also, 50 alloys mean infinite alloys. Its too much.

Reply:
Irrelevant. OF COURSE the Harvesting Marvel has infinite alloys. It was built by some Fallen Empire to extract- you guessed it -infinite Minerals and put them into infinite Alloys. The player cannot play the Harvesting Marvel and the Harvesting Marvel ultimately becomes a Crisis. It is only logical that the Marvel _actually serve as a threat_.

Reply:
It have minerals at starting systems